### PR TITLE
Add benchmark for DrawDRRect

### PR DESCRIPTION
**Pull request by gw280**
Add a new DrawDRRect benchmark. This just mirrors the DrawRRect benchmark but uses a second rrect that's inset by 10% in all four corners to generate the drrect.

### CODE REVIEW

File: display_list/display_list_benchmarks.cc [hunk 1]
```diff
@@ -335,6 +335,86 @@ void BM_DrawRRect(benchmark::State& state,
...
+  SkRRect rrect, rrect_2;
+
+  SkVector set_radii[4];
+  for (size_t i = 0; i < 4; i++) {
+    set_radii[i] = radii[i] * multiplier;
+  }
+  rrect.setRectRadii(SkRect::MakeLTRB(0, 0, length, length), set_radii);
+
+  for (size_t i = 0; i < kRRectsToDraw; i++) {
+    rrect.inset(0.1f * length, 0.1f * length, &rrect_2);
```
Comment (flar) [inline display_list/display_list_benchmarks.cc:395]: The dx and dy are also added(subtracted) to/from the radii. I think the smallest radius you have is length/8? If so then you might just barely squeak by on not creating a degenerate RRect. Do I read that right?

**Review by flar — APPROVED** [review]
I think it is fine as long as you double check the math on insetting the inner RRect to make sure it doesn't degenerate.

File: display_list/display_list_benchmarks.cc (hunk 1 above)
Comment (gw280) [reply display_list/display_list_benchmarks.cc:395]: The smallest radius is 2.0f in the nine patch rrect. In the smallest rect we request (16px length) that makes the multiplier 1.0 so the smallest absolute radius is 2.0f.

That makes our rrect degenerate but only in one corner, so we won't be just hitting some optimisation to call drawRect instead. I think it's fine as is, as I can imagine a usecase where one corner is squared off is plausible.

Comment (flar) [reply display_list/display_list_benchmarks.cc:395]: If we test squashed corners, then they should be tested in their own benchmarks, not mixed into the results of another.

Comment (gw280) [reply display_list/display_list_benchmarks.cc:395]: Changed the radii to larger numbers.

File: display_list/display_list_benchmarks.cc [hunk 2]
```diff
@@ -366,7 +366,7 @@ void BM_DrawDRRect(benchmark::State& state,
       radii[3] = SkVector::Make(5.0f, 5.0f);
       break;
     case SkRRect::Type::kNinePatch_Type:
-      radii[0] = SkVector::Make(5.0f, 2.0f);
+      radii[0] = SkVector::Make(5.0f, 7.0f);
       radii[1] = SkVector::Make(3.0f, 2.0f);
```
Comment (flar) [inline display_list/display_list_benchmarks.cc:370]: missed one

**Pull request merged**